#
# Submitted-By: Robert Schwebel, 2005-05-08
# Committed-By: Robert Schwebel
#
# Error: 
#
# config.sub claims that this is no valid configuration. 
#
# Description: 
#
# config.sub claims that this is no valid configuration. 
#
# State:
# 
# unknown
#

diff -urN module-init-tools-3.2-pre1/config.sub module-init-tools-3.2-pre1-ptx/config.sub
--- module-init-tools-3.2-pre1/config.sub	2002-11-26 02:38:55.000000000 +0100
+++ module-init-tools-3.2-pre1-ptx/config.sub	2005-05-08 13:34:24.000000000 +0200
@@ -281,10 +281,10 @@
 	  basic_machine=$basic_machine-pc
 	  ;;
 	# Object if more than one company name word.
-	*-*-*)
-		echo Invalid configuration \`$1\': machine \`$basic_machine\' not recognized 1>&2
-		exit 1
-		;;
+#	*-*-*)
+#		echo Invalid configuration \`$1\': machine \`$basic_machine\' not recognized 1>&2
+#		exit 1
+#		;;
 	# Recognize the basic CPU types with company name.
 	580-* \
 	| a29k-* \
@@ -1099,7 +1099,7 @@
 	# The portable systems comes first.
 	# Each alternative MUST END IN A *, to match a version number.
 	# -sysv* is not here because it comes later, after sysvr4.
-	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
+	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* | -uclibc* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -solaris* | -sym* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
